src/client/opamCommands.ml

```diff
@@ -3616,8 +3616,27 @@ let source cli =
       "Choose package without consideration for \
        the current (or any other) switch (installed or pinned packages, etc.)"
   in
-  let source global_options atom dev_repo pin no_switch dir () =
+  let no_checksums =
+    mk_flag ~cli (cli_from cli2_2) ["no-checksums"]
+      "Do not verify the checksum of downloaded source.\
+       This is equivalent to setting $(b,\\$OPAMNOCHECKSUMS) to \"true\"."
+  in
+  let req_checksums =
+    mk_flag ~cli (cli_from cli2_2) ["require-checksums"]
+      "Enforce checksum verification befor downloading sources.\
+       This is equivalent to setting $(b,\\$OPAMREQUIRECHECKSUMS) to \"true\"."
+  in
```

Comment:
might be worth sharing the flags definitions with the ones used in `OpamArg.build_options` ? (maybe that was too tedious to be worth it, in that case it's alright)

Reply:
It is possible to share it easily with `build_options`, (cf. d69eb84ee125643fce1a1dd0f344c2c5d539f5ad).
I've tried to move them from build options to global options, but we loose the ability to notify that some new commands now permit and use `--*-checksum` options (no possibility to describe cli validity fine grained). Cf ef3b1e2a149404163eec26d28e7d7b9247fc9cb0